src/agent/reqwest-retry/src/lib.rs

```diff
@@ -169,19 +169,20 @@ impl SendRetry for reqwest::RequestBuilder {
     }
 }
 
-#[cfg(test)]
-mod test {
-    use super::*;
+// TODO: convert to feature-gated integration test.
```

Comment:
I think we need to make this an integration test, and we should document the expected behavior of the test endpoint.